index.html

```diff
@@ -4304,7 +4308,7 @@ <h3>Device Theft and Impersonation</h3>
         </li>
         <li>
 Enabling password, biometric, or multi-factor authentication for the
-<a>credential</a> repository.
+<a>verifiable credential</a> <a>repository</a>.
```

Comment:
nope, credential. Not all credentials stored in a repository will be verifiable.

Reply:
Fixed.